client/landing/stepper/hooks/use-site-transfer/query.ts

```diff
@@ -52,14 +52,22 @@ export function getSiteTransferStatusQueryKey( siteId: number ) {
 }
 
 const shouldRefetch = ( status: TransferStates | undefined ) => {
-	if ( ! status || status === transferStates.NONE ) {
+	if ( ! status ) {
 		return false;
 	}
 
+	// This condition ensures that when the status is NONE,
+	// we expect it might change in another tab. This allows the UI to reflect any updates dynamically.
```

Comment:
Why do we expect this?

Reply:
The full explanation is [here](https://github.com/Automattic/wp-calypso/pull/93789#discussion_r1814884553).
But the initial main goal of all this tasks around "Activate now" button was - we should avoid rendering it always, to avoid errors at the backend. So I covered all possible cases with it.

Reply:
Gotcha. Right now we are also refetching for sites that can't go Atomic, though. Is there a way we can prevent refetching for sites on the Premium plan, for example?